chess/variant.py

```diff
@@ -727,6 +727,13 @@ def legal_drop_squares_mask(self) -> chess.Bitboard:
             return chess.BB_EMPTY
 
     def legal_drop_squares(self) -> chess.SquareSet:
+        """
+        Get the squares where a legal drop is possible for the side to move.
```

Comment:
It should say "Gets the squares..." and not "Get the squares...". It's a description, so it must be worded like that.